docs/.vitepress/en.ts

```diff
@@ -0,0 +1,117 @@
+import { defineConfig } from 'vitepress'
+
+export const en = defineConfig({
+  description: "Consume OpenAPI 3.0 & 3.1 schemas in TypeScript",
+  themeConfig: {
+    nav: [
+      {
+        text: "Version",
+        items: [
+          { text: "7.x", link: "/introduction" },
+          { text: "6.x", link: "/6.x/introduction" },
+        ],
+      }
+    ],
+    sidebar: {
+      // 6.x docs
+      "/6.x/": [
+        {
+          text: "openapi-typescript (6.x)",
+          items: [
+            { text: "Introduction", link: "/6.x/introduction" },
+            { text: "CLI", link: "/6.x/cli" },
+            { text: "Node.js API", link: "/6.x/node" },
+            { text: "Advanced", link: "/6.x/advanced" },
+            { text: "About", link: "/6.x/about" },
+          ],
+        },
+        {
+          text: "openapi-fetch",
+          items: [
+            { text: "Getting Started", link: "/openapi-fetch/" },
+            {
+              text: "Middleware & Auth",
+              link: "/openapi-fetch/middleware-auth",
+            },
+            { text: "Testing", link: "/openapi-fetch/testing" },
+            { text: "Examples", link: "/openapi-fetch/examples" },
+            { text: "API", link: "/openapi-fetch/api" },
+            { text: "About", link: "/openapi-fetch/about" },
+          ],
+        },
+      ],
+      // 🌐
+      "/zh/": [
```

Comment:
We can probably remove `zh` from the English sidebar, right?

Reply:
Yes! I've removed it.